query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/schema/AlterTypeAddField.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.api.querybuilder.schema;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.api.core.type.DataType;
+import com.datastax.oss.driver.api.querybuilder.BuildableQuery;
+
+public interface AlterTypeAddField {
+  /** Completes ALTER TYPE by adding a field definition in the ALTER TYPE statement. */
+  BuildableQuery addField(CqlIdentifier fieldName, DataType dataType);
```

Comment:
Any reason why we can't have those methods directly in `AlterTypeStart`? It looks the same as `alterField`.

Reply:
Hrmm, yeah that looks like an oversight, maybe originally when it was implemented the thought was you would be able to add multiple fields in an alter (like rename) but that isn't the case.